static/snippet.js

```diff
@@ -3,15 +3,16 @@ import ReactDOM from 'react-dom';
 
 class Snippet extends React.Component {
     render() {
-        if (!this.props.spans || this.props.spans.length == 0) {
+        const { sans } = this.props;
```

Comment:
typo here - I expect this should be `spans` instead of `sans` and `this.props.spans`, not `this.props`?